packages/manager/modules/octavia-load-balancer/src/load-balancers/load-balancer/pools/detail/health-monitor/module.js

```diff
@@ -6,13 +6,14 @@ import service from './service';
 
 import create from './create';
 import edit from './edit';
+import deletion from './delete';
```

Comment:
```suggestion
import delete from './delete';
```

Reply:
delete is a javascript reserved word so better use deletion